aws/resource_aws_cloudtrail.go

```diff
@@ -72,6 +73,46 @@ func resourceAwsCloudTrail() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validateArn,
 			},
+			"event_selector": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 5,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"read_write_type": &schema.Schema{
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringInSlice([]string{"All", "ReadOnly", "WriteOnly"}, false),
+						},
+
+						"include_management_events": &schema.Schema{
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  true,
+						},
+
+						"data_resource": &schema.Schema{
+							Type:     schema.TypeSet,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"type": &schema.Schema{
+										Type:         schema.TypeString,
+										Required:     true,
+										ValidateFunc: validation.StringInSlice([]string{"AWS::S3::Object"}, false),
```

Comment:
CloudTrail as of recently also supports Lambda, so you might want to add "AWS::Lambda::Function" to the list of allowed types.

Reply:
Thanks comment!
I added Lambda.